app/code/core/Mage/Wishlist/Model/Resource/Item/Collection.php

```diff
@@ -22,6 +23,11 @@
  */
 class Mage_Wishlist_Model_Resource_Item_Collection extends Mage_Core_Model_Resource_Db_Collection_Abstract
 {
+    /**
+     * Collection wishlist instance
+     */
+    protected Mage_Wishlist_Model_Wishlist $_wishlist;
```

Comment:
the comment says it's a collection but the type is not a collection. also are we supposed to not use $_ anymore?